lib/zone.ts

```diff
@@ -136,11 +136,11 @@
 interface Zone {
   /**
    *
-   * @returns {Zone} The parent Zone.
+   * @returns {?Zone} The parent Zone.
```

Comment:
Thank for the review, I am new to closure compiler, so I just check the document and add the type I can find.

So I just revert all those inline type changes from zone.ts, is that correct?